### PR TITLE
fixes #3279 roles edit page is long and missing a filter

### PR DESCRIPTION
**Pull request by abenari**
take 2

### CODE REVIEW

**Comment by ohadlevy** [comment]
test failures

**Comment by abenari** [comment]
[test]

**Comment by domcleal** [comment]
It failed pretty consistently, I wouldn't think a retest will fix it.

**Comment by abenari** [comment]
No failures when I run the test on my machine, this is probably a CI issue.

**Comment by domcleal** [comment]
It's not just CI...

<pre>
[dcleal@cobalt foreman]$ git co abenari/roles2
HEAD is now at 5d17520... fixes #3279 roles edit page is long and missing a filter
[dcleal@cobalt foreman]$ ruby -Itest test/functional/roles_controller_test.rb 
--------------------------------------------------------------------------------
RR deprecation warning: RR now has an autohook system. You don't need to
`include RR::Adapters::*` in your test framework's base class anymore.
--------------------------------------------------------------------------------
Run options: --seed 62348
# Running tests:
...E.E..E.
Finished tests in 1.686340s, 5.9300 tests/s, 13.6390 assertions/s.
  1) Error:
RolesControllerTest#test_get_edit:
ActionView::Template::Error: undefined method `humanize' for nil:NilClass
    /home/dcleal/code/foreman/foreman/app/views/roles/_form.html.erb:19:in `block (3 levels) in _app_views_roles__form_html_erb___3761336411927021337_90339080'
[snip for brevity]
10 tests, 23 assertions, 0 failures, 3 errors, 0 skips
Coverage report generated for Functional Tests to /home/dcleal/code/foreman/foreman/coverage. 5624 / 11570 LOC (48.61%) covered.
</pre>

**Comment by abenari** [comment]
@domcleal tests are now passing :)

File: app/views/roles/_form.html.erb [hunk 1]
```diff
@@ -5,24 +5,35 @@
 
   <%= text_f f, :name, :class => @role.builtin? ? "disabled" : ""  %>
 
+  <div class='control-group'>
+    <label class='control-label'><%= _('Permissions') %></label>
+    <input placeholder="<%= _('Filter permissions') %>" class="search-query" onkeyup="filter_permissions(this);" type='text'>
```
Comment (ohadlevy) [inline app/views/roles/_form.html.erb:10]: search is case sensitive, imho it should be insensative

**Comment by abenari** [comment]
[test]

**Comment by ohadlevy** [comment]
merged as e3a220b, thanks!

**Pull request closed without merging**